manim/utils/color/core.py

```diff
@@ -1021,6 +1021,9 @@ def __xor__(self, other: Self) -> Self:
             self._internal_from_integer(self.to_integer() ^ int(other), 1.0)
         )
 
+    def __hash__(self) -> str:
```

Comment:
The Python docs recommend that `__hash__` returns an integer.
Can you just return `hash(self.to_hex(with_alpha=True))`?

Reply:
Don't forget to change the return typehint to `int`.